skins/cat17/faq-page/.gitignore

```diff
@@ -0,0 +1,13 @@
+.DS_Store
```

Comment:
Is this file still necessary? It probably was generated and everything in here should be covered by the parent .gitignore.